client/components/domains/registrant-extra-info/uk-form.jsx

```diff
@@ -75,19 +77,24 @@ export class RegistrantExtraInfoUkForm extends React.PureComponent {
 			return;
 		}
 
-		this.props.updateContactDetailsCache( {
+		const payload = {
 			extra: {
 				uk: pick( defaultValues, neededRequiredDetails ),
 			},
-		} );
+		};
+
+		this.props.updateContactDetailsCache( payload );
+		this.props?.onContactDetailsChange( payload );
```

Comment:
Same as before, `this.props` will exist; you probably meant to mark `onContactDetailsChange` as optional here.